src/Monitor/Monitor.js

```diff
@@ -56,93 +56,95 @@ class Monitor extends Component {
       }
     };
   }
-  areTasksValid = (tasks) => (tasks && tasks.length > 0)
-  handleClickCenterButton = () => {
+  areTasksValid(tasks) {
+    return tasks && tasks.length > 0;
+  }
+  handleClickCenterButton() {
```

Comment:
Avec cette notation de fonction, il me semble qu'il est nécessaire de bind this dans le constructeur. Sinon this est undefined est la fonction plante à l'execution

Reply:
yes c'est ce que j'ai fait :)

Reply:
https://github.com/theodo/workflow-monitor/pull/5/files#diff-b3e25b8a44cf88c5a7af82c2c6985e42R163

Reply:
Tu ne l'as pas fait dans ce composant

Reply:
`this.handleClickCenterButton = this. handleClickCenterButton.bind(this);`

Reply:
je sais pas trop quoi te dire ca marche bien pourtant